GitCommands/DiffMergeTools/Kdiff3.cs

```diff
@@ -7,6 +7,9 @@ internal class Kdiff3 : DiffMergeTool
         /// <inheritdoc />
         public override string ExeFileName => "kdiff3.exe";
 
+        /// <inheritdoc />
+        public override string MergeCommand =>  \"$BASE\" \"$REMOTE\" "\"$LOCAL\" -o \"$MERGED\"";
```

Comment:
```suggestion
        public override string MergeCommand =>  "\"$BASE\" \"$REMOTE\" \"$LOCAL\" -o \"$MERGED\"";
```

Reply:
Are you suggesting the PR should have correct syntax?
I have a fever and my brain is only partially working...

Reply:
Yes, as you already have force-pushed.
Beterschap!